src/site/includes/header.html

```diff
@@ -12,6 +12,10 @@
 <meta name="MobileOptimized" content="320">
 <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
+  {% if buildtype != 'vagovprod' %}
```

Comment:
Gah I apologize for the indentation on this. After saving with prettier the entire file was completely updated so I only grabbed these 4 lines out of the diff.